src/types/teams.ts

```diff
@@ -6,7 +6,7 @@ export enum TeamName {
   community = 'Community',
   design = 'Design',
   devOps = 'DevOps',
-  discordManagers = 'Discord Managers',
+  discordManagers = 'Discord',
```

Comment:
can you also update the key to be `discord`?

Reply:
Updated